packages/editor/src/components/editor-interface/index.js

```diff
@@ -214,18 +215,22 @@ export default function EditorInterface( {
 				)
 			}
 			actions={
-				! isPreviewMode ? (
-					<SavePublishPanels
-						closeEntitiesSavedStates={ closeEntitiesSavedStates }
-						isEntitiesSavedStatesOpen={
-							entitiesSavedStatesCallback
-						}
-						setEntitiesSavedStatesCallback={
-							setEntitiesSavedStatesCallback
-						}
-						forceIsDirtyPublishPanel={ forceIsDirty }
-					/>
-				) : undefined
+				! isPreviewMode
+					? customSavePanel || (
```

Comment:
I'm guessing these two conditions can be merged using a `&&`

Reply:
They could, but it would be like ` !x && a || b` and because of the `||` I think it's a bit clearer this way..